javascript-data-model/main.js

```diff
@@ -0,0 +1,73 @@
+/* exported orderHistory */
+var orderHistory = [
+
+  {
+    orderedItems: [
```

Comment:
This seems like it's just an array of all items ordered in the past, but not organized by order. Try to revise this to show each individual order separately, and the items that were in that order listed in each order.